pyladies_cz.py

```diff
@@ -115,8 +115,9 @@ def read_yaml(filename):
         if 'description' in lesson:
             lesson['description'] = convert_markdown(lesson['description'],
                                                      inline=True)
-        for mat in lesson['materials']:
-            mat['name'] = convert_markdown(mat['name'], inline=True)
+        if 'materials' in lesson:
+            for mat in lesson['materials']:
```

Comment:
Mně vždycky přišlo lepší v takovýchto případech použít `for mat in lesson.get('materials', ()):`